Model/Import/Category.php

```diff
@@ -1298,7 +1298,7 @@ protected function uploadMediaFiles($fileName)
     {
         try {
             $res = $this->getUploader()->move($fileName);
-            return $res['file'];
+            return '/media/catalog/category' . $res['file'];
```

Comment:
It is rather strange that this is needed as this should be handled already:

https://github.com/firegento/FireGento_FastSimpleImport2/blob/1f791a491dbeb06ed23a6e00712a8492b887e789/Model/Import/Category.php#L1331-L1333

Reply:
It's not on my side, could you try?
This is what I have:
![](https://screen.ph2m.fr/screen_c6734d20.png)